src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -9,47 +9,15 @@
         </div>
```

Comment:
Silly question, but is this component eventually going to be discontinued/replaced or will it still serve some function?

Reply:
I will serve as Review inputs where the charts will be displayed.